server.go

```diff
@@ -883,6 +901,9 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
+		if handled := s.processUnknownStreamHandler(t, stream, trInfo); handled {
```

Comment:
Yeah, but then it is unclear what to do with `dummyService` service desc. Moving it to the top of the file would pollute it with something ambiguous.

I think that keeping this in a separate function with a clear name of what it does makes sense, given that this function is already fairly busy. Unless you guys have a policy of keeping the number of functions low.